app/src/main/java/com/ataulm/whatsnext/Actor.kt

```diff
@@ -1,3 +1,7 @@
 package com.ataulm.whatsnext
 
-data class Actor(val character: String?, val person: Person)
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class Actor(val character: String?, val person: Person): Parcelable
```

Comment:
Lets us pass this stuff in intents. Maybe it would have been better to mark these all as Serializable instead - that way we wouldn't be using Android stuff in what should be domain classes.